### PR TITLE
[#9115] Suggestion for student feedback pageminor UI overflow issue

### LINKED ISSUE #9115: Suggestion for student feedback page minor UI overflow issue
Under question 5 of the student feedback page, each of the dropdown and table rows are displayed like this: (notice the overflow of the table on the right)
image

The html code is as follows
image

**Additional info**
Suggested simple changes:
- Set parent `<div>` classes: `col-sm-12 col-md-10 col-md-offset-1`
- Set children `<div>`s classes: `form-group`
- Remove "(Student):" text before the `<label>` tag
- Set the text of the default selected `<option>` to "Select a student" or something informative

image

Once changes are applied, it should look like the top one of the following image:
image

### PR DESCRIPTION
**Pull request by css911**
[#9115] Suggestion for student feedback pageminor UI overflow issue.

1. The first part of the issue was to stop the overflow of the rubric question.

2. The Second part of the issue was to change the tooltip of the student feed back form

Fixes #9115  #8542

**Outline of Solution**

 I solved the first issue by adding css to the teammatesCommon.css regarding the display and overflow-x attribute related to the table by setting display to block and overflow-x to auto.

  I solved this issue of changing the tooltip by changing the FEEDBACK_SESSION_EDIT_SAVE string to "You can edit your responses after submission at any time before the deadline"

I am also adding the html files as the changes in them will be helpful in the testing. 

There was actually repetition of the issue which were reported for the rubric question overflow. the #8542 and #9115 are talking about the same issue.

### CODE REVIEW

**Comment by wkurniawan07** [comment]
@css911, the first part of the issue concerns rubric question type. Your screenshot shows text/essay question type.

**Comment by css911** [comment]
image

Is this the correct place where is should look for @wkurniawan07.
Sir,but according the the screenshot which was given while reporting the issue it also shows a drop down menu for selecting the students.

https://user-images.githubusercontent.com/7817101/45229695-2a3c5d00-b2f9-11e8-9a55-a8b819e32bf0.png

Or is there a condition which disables the drop down menu and statically creates the rubric question feedback when the students are less.

**Comment by css911** [comment]
@wkurniawan07 I have tried certain changes to make the rubric question scroll horizontally.

The overload issue before.

before 

after the changes are done.

after

If the changes are ok should i  create a new PR with both changes

**Comment by css911** [comment]
@wkurniawan07 Sir, I am done from my side on this issue and I am waiting for the reviews till the mean time can i take to solve any other issue.

File: src/main/webapp/stylesheets/teammatesCommon.css [hunk 1]
```diff
@@ -1604,3 +1604,8 @@ Prevent Long URL From Breaking Out of Container
 .icon-info::before {
     content: '\e086';
 }
+
+.table {
```
Comment (wkurniawan07) [inline src/main/webapp/stylesheets/teammatesCommon.css]: This doesn't look safe, i.e. all other elements with class `table` will also get styled like this which may not be what we want. Why not follow the suggestions in the issue report of #9115?

File: src/test/resources/pages/instructorCoursesAddDupIdFailed.html [hunk 2]
```diff
@@ -112,7 +112,7 @@ <h2>
             Software Engineering $^&*()
           </td>
           <td data-date-stamp="${datetime.now.iso8601utc}" data-original-title="${datetime.now}" data-toggle="tooltip" id="coursecreateddate2">
-            ${datetime.now.courses}
+             ${datetime.now.courses}
```
Comment (wkurniawan07) [inline src/test/resources/pages/instructorCoursesAddDupIdFailed.html]: Unnecessary change like this (and in many other `*.html` files) should be reverted.

File: src/main/webapp/stylesheets/teammatesCommon.css (hunk 1 above)
Comment (css911) [reply src/main/webapp/stylesheets/teammatesCommon.css]: Sure sir!. I will get it done accordingly. Thank you for suggesting changes. I will close this pull request and create clean PR.

Comment (css911) [reply src/main/webapp/stylesheets/teammatesCommon.css]: @wkurniawan07, Sir i tried out the changes as suggested in PR #9115 

before performing changes.
before
(code in inspect element is the code where the changes were suggested.) 

after performing the suggested changes.
after1

The changes are able to get the rubric question tab below the pull down menu.

Comment (css911) [reply src/main/webapp/stylesheets/teammatesCommon.css]: @wkurniawan07  Sir, I carried out the changes as mentioned in PR #9115.

Rubric Question before changes
before
(The code in the inspect element tab was suggested to be changed.)

Rubric Question after changes
after1
(The code in the inspect element is the modified code)

Please note: The problem of overflow still exits. Even if the table now is appearing under the pull down menu

Sir as correctly pointed out by you.
The  css code for the table will effect every class which has class name table.

Sir, but the effect of this css code is not observed on tables which are not overflowing. as the overflow only in x direction is set to auto and the display is of block nature.

.table{
display: block;
overflow-x: auto;
}

Sir, can you please suggest a better direction for solving this problem.

Thank you sir.

**Comment by madanalogy** [comment]
Closing due to inactivity, and it seems that the original repo of this PR is no longer available
